index.html

```diff
@@ -180,6 +187,45 @@ <h3>Skills</h3>
     </section>
 
   </section>
+
+  <section class="contact-page">
+
+    <h2 class="contact-comment">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+  
+    <form action="https://formspree.io/f/mqkoaqdk" method="post">
```

Comment:
- [ ] I kindly suggest that you follow the figma design very closely by ensuring:
  - The contact form aligned to the center because on yours there's a lot of margin to the right of the contact form.
  - Kindly remove the black borders on the inputs that appear when typing.
  - Kindly add the horizontal bar underneath the social media icons.

| Your Design  | Figma Design |
| ------------- | ------------- |
|  <img width="208" alt="image" src="https://user-images.githubusercontent.com/52934128/213128216-ee04fb13-2110-4acd-82e2-18706e101435.png">| <img width="180" alt="image" src="https://user-images.githubusercontent.com/52934128/213128460-cb25c48d-8edb-4b9a-be0a-5d6b49d2d967.png">|